hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java

```diff
@@ -376,6 +376,50 @@ public void addDependencies(ServiceTarget target, ServiceBuilder<MonitorService>
         // Note that if we are running in host controller or similiar, JNDI binding is not available.
         String jndiName = bootConfiguration.getApiJndi();
         boolean bindJndi = (jndiName == null || jndiName.isEmpty() || processType.isManagedDomain()) ? false : true;
+        /**
+         * Currently, when trying to re-bind the jndi it triggers... Need to work this out when everything else is done.
+         12:59:09,467 ERROR [org.jboss.as.controller.management-operation] (management-handler-thread - 2) WFLYCTL0013: Operation ("write-attribute") failed - address: ([("subsystem" => "hawkular-wildfly-agent")]): org.jboss.msc.service.DuplicateServiceException: Service jboss.naming.context.java.global.hawkular.agent.api is already registered
```

Comment:
When below code is executed (specifically `binderBuilder.install();`) it throws an exception.
I suppose we could unbind this when calling the remove operation or verify here if is already there before installing.